krux_cloud_health/cloud_health.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+"""
+Class to retrieve data from Cloud Health Tech API
+"""
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+import sys
+import urlparse
+
+#
+# Third party libraries
+#
+
+import requests
+
+#
+# Internal libraries
+#
+
+from krux.cli import get_parser, get_group
+
+
+API_ENDPOINT = "https://apps.cloudhealthtech.com/"
+NAME = "cloud-health-tech"
+
+
+def add_cloud_health_cli_arguments(parser):
+    # Add those specific to the application
+    group = get_group(parser, NAME)
+    
+    group.add_argument(
+        'api_key',
+        type=str,
+        help="API key to retrieve data",
+    )
+
+
+def get_cloud_health(args, logger, stats):
+    if not args:
+        parser = get_parser(description=NAME)
+        add_dns_cli_arguments(parser)
+        args = parser.parse_args()
+
+    if not logger:
+        logger = get_logger(name=NAME)
+
+    if not stats:
+        stats = get_stats(prefix=NAME)
+
+    return CloudHealth(
+        api_key=args.api_key,
+        logger=logger,
+        stats=stats,
+        )
+
+
+class CloudHealth(object):
+    """
+    """
+    def __init__(self, api_key, logger, stats):
+        self.api_key = api_key
+        self.logger = logger
+        self.stats = stats
+
+    def costHistory(self, month_input=None):
```

Comment:
I'm gonna say I don't like camelCase, you use functions_with_underscores, why the mix?